language/move-vm/transactional-tests/tests/builtins/vector_ops_destroy_non_empty.mvir

```diff
@@ -0,0 +1,10 @@
+//# run
```

Comment:
Maybe have another script in this file that succeeds? Maybe even one with a type without drop if you want to get fancy :)

Reply:
The one without `drop` ability should be caught by the bytecode verifier I guess? As tested in this test case: https://github.com/diem/diem/blob/main/language/bytecode-verifier/transactional-tests/tests/locals_safety/vector_ops_non_droppable_resource_not_destroyed.mvir